app/src/androidTest/java/org/cimbar/camerafilecopy/ExampleInstrumentedTest.java

```diff
@@ -1,27 +0,0 @@
-package org.cimbar.camerafilecopy;
```

Comment:
Removing unused test stubs. (they were breaking the build)

https://github.com/sz3/cfc/pull/48#discussion_r2294925853